scss/components/_case-list-table.scss

```diff
@@ -0,0 +1,139 @@
+.civicase__table--caselist {
+  .fixed-col {
+    .label {
+      font-size: 0.8em;
+    }
+  }
+
+  .crm_notification-badge {
+    line-height: 18px;
+    padding: 0 10px;
+  }
+
+  &__header {
+    &.affix {
+      display: block;
+      left: 0;
+      margin-left: 310px;
+      overflow-x: hidden;
+      overflow-y: visible;
+      right: 0;
+      top: 60px;
+      z-index: 9999;
+
+      tr {
+        display: table;
+        width: 100%;    
+      }
+
+      th {
+        display: table-cell;
+      }      
+    }
+  }
+}
+
+.case-list-panel:not(.viewing-case) {
+  .outer {
+    background: #fff;
+    position: relative;
+  }
+
+  .inner {
+    margin-left: 310px;
+    overflow-x: auto;
+    overflow-y: visible;
+  }
+
+  .custom-scroll-wrapper {
+    border: 0;
+    margin-left: 310px;
+    overflow-x: scroll;
+    overflow-y: hidden;
+    width: calc(100% - 310px);
+
+    .custom-scroll {
```

Comment:
What is the usage of this class?
and all class names needs to be converted to BEM.

Reply:
Done

Reply:
Done